ru/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -1,53 +1,53 @@
-<!-- Entities for AccountWizard -->
+﻿<!-- Entities for AccountWizard -->
 
-<!ENTITY windowTitle.label "Мастер учётных записей">
+<!ENTITY windowTitle.label "Мастер учетных записей">
 <!ENTITY accountWizard.size "width: 50em; height: 40em;">
 <!ENTITY macAccountWizard.size "width: 42em; height: 40em;">
 
 <!-- Entities for Account Type page -->
 
-<!ENTITY accountSetupInfo.label "Чтобы получать сообщения, сначала нужно создать учётную запись почты или новостей.">
-<!ENTITY accountTypeTitle.label "Параметры новой учётной записи">
-<!ENTITY accountTypeDesc.label "Мастер учётных записей соберёт информацию, необходимую для создания учётной записи почты или новостей. Если вы не знаете всей необходимой информации, свяжитесь с вашим системным администратором или поставщиком услуг Интернета.">
-<!ENTITY accountTypeDirections.label "Выберите, какой тип учётной записи вы хотите создать:">
-<!ENTITY accountTypeMail.label "Учётная запись электронной почты">
+<!ENTITY accountSetupInfo.label "Для получения сообщений сначала необходимо настроить учетную запись почты или группы новостей.">
+<!ENTITY accountTypeTitle.label "Настройка новой учетной записи">
+<!ENTITY accountTypeDesc.label "Этот мастер соберет информацию, необходимую для настройки учетной записи почты или группы новостей. Если необходимая информация неизвестна, обратитесь к системному администратору или поставщику услуг Интернета.">
+<!ENTITY accountTypeDirections.label "Выберите тип учетной записи, которую необходимо настроить:">
+<!ENTITY accountTypeMail.label "Учетная запись электронной почты">
 <!ENTITY accountTypeMail.accesskey "m">
-<!ENTITY accountTypeNews.label "Учётная запись новостей">
+<!ENTITY accountTypeNews.label "Учетная запись группы новостей">
 <!ENTITY accountTypeNews.accesskey "w">
 
 <!-- Entities for Identity page -->
 
 <!ENTITY identityTitle.label "Персональные данные">
-<!ENTITY identityDesc.label "Для каждой учётной записи имеются персональные данные, которые смогут увидеть читающие ваши письма адресаты.">
+<!ENTITY identityDesc.label "Для каждой учетной записи указываются персональные данные, которые смогут увидеть получатели ваших сообщений. ">
 
-<!ENTITY fullnameLabel.label "Выводимое имя:">
+<!ENTITY fullnameLabel.label "Ваше имя:">
 <!ENTITY fullnameLabel.accesskey "Y">
 
-<!ENTITY emailLabel.label "Адрес электронной почты:">
+<!ENTITY emailLabel.label "Адрес эл. почты:">
 <!ENTITY emailLabel.accesskey "E">
-<!ENTITY emailDesc.label "Введите ваш адрес электронной почты, куда другие будут отправлять вам сообщения">
+<!ENTITY emailDesc.label "Введите ваш адрес электронной почты, который будет использоваться для отправки вам сообщений">
```

Comment:
CORRECTIONS NEEDED for line 28. Let's skip 3 unneded words, since user just saw them and use: +<!ENTITY emailDesc.label "Введите адрес, который будет использоваться для отправки вам сообщений">